components/libc/compilers/common/ctime.c

```diff
@@ -115,6 +115,49 @@ static rt_err_t _control_rtc(int cmd, void *arg)
 #endif /* RT_USING_RTC */
 }
 
+/* lightweight timezone and daylight saving time */
+#ifdef RT_LIBC_USING_LIGHT_TZ_DST
+#ifndef RT_LIBC_TZ_DEFAULT_HOUR
+#define RT_LIBC_TZ_DEFAULT_HOUR   (8U)
+#endif /* RT_LIBC_TZ_DEFAULT_HOUR */
+
+#ifndef RT_LIBC_TZ_DEFAULT_MIN
+#define RT_LIBC_TZ_DEFAULT_MIN    (0U)
+#endif /* RT_LIBC_TZ_DEFAULT_MIN */
+
+#ifndef RT_LIBC_TZ_DEFAULT_SEC
+#define RT_LIBC_TZ_DEFAULT_SEC    (0U)
+#endif /* RT_LIBC_TZ_DEFAULT_SEC */
+
+static volatile int32_t _current_timezone_sec = \
+    RT_LIBC_TZ_DEFAULT_HOUR * 3600U + RT_LIBC_TZ_DEFAULT_MIN * 60U + RT_LIBC_TZ_DEFAULT_SEC;
+
+/* return current timezone offset in seconds */
+void rt_tz_set(int8_t hour, int8_t min, int8_t sec)
```

Comment:
```suggestion
void rt_tz_set(int32_t timezone_offset_sec)
```
是不是可以和 rt_tz_get 一样，都使用 timezone_offset_sec